tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/finite/FiniteTypeTest.java

```diff
@@ -367,4 +367,21 @@ public void testEscapedTypeName() {
         BValue[] returns = BRunUtil.invoke(result, "testEscapedTypeName");
         Assert.assertEquals(returns[0].stringValue(), "-");
     }
+    @Test()
+    public void testFiniteTypesWithPositiveIntegers() {
+        BValue[] returns = BRunUtil.invoke(result, "testFiniteTypesWithPositiveIntegers");
+        Assert.assertEquals(returns.length, 1);
+        Assert.assertNotNull(returns[0]);
+        Assert.assertTrue(returns[0] instanceof BInteger);
+        Assert.assertEquals(((BInteger) returns[0]).intValue(), 5);
+    }
+    @Test()
+    public void testFiniteTypesWithPositiveFloats() {
+        BValue[] returns = BRunUtil.invoke(result, "testFiniteTypesWithPositiveFloats");
+        Assert.assertEquals(returns.length, 1);
+        Assert.assertNotNull(returns[0]);
+        Assert.assertTrue(returns[0] instanceof BFloat);
+        Assert.assertEquals(((BFloat) returns[0]).floatValue(), 1.5);
+    }
+
```

Comment:
We usually try to do the testing on ballerina itself. And panic if not the intended result of the test.

Reply:
+1

Reply:
Done